.github/workflows/autofix.yml

```diff
@@ -27,10 +27,11 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version-file: .github/node-version.txt
-      - name: Run prettier
-        run: |
-          npm ci
-          npm run prettier
+      - run: npm ci
+        working-directory: web
+      - run: npx eslint --fix .
+        working-directory: web
+      - run: npx prettier --write .
```

Comment:
I personally would use `npm run prettier` (and add a command for eslint as well) so that this config does not need to know any details (arguments etc.).